src/swagger/specGenerator2.ts

```diff
@@ -62,7 +63,16 @@ export class SpecGenerator2 extends SpecGenerator {
         };
 
         if (referenceType.additionalProperties) {
-          definitions[referenceType.refName].additionalProperties = this.buildAdditionalProperties(referenceType.additionalProperties);
+            definitions[referenceType.refName].additionalProperties = this.buildAdditionalProperties(referenceType.additionalProperties);
```

Comment:
hm.. there are four spaces here vs. two spaces in the rest of the file?

Reply:
I can fix it. I’d really like us to switch over to `prettier` since it would fix this for us maintainers. Would you be open to me migrating us to prettier after this PR merged? If I do it as part of this PR it will light up the entire repository with changes and then it won’t be possible to see what was substantively changed.

Reply:
Uh, sure. I've never used it. Isn't linting able to tell us about this anyway?

Reply:
Prettier formats the code for you. So we could remove the three annoying longer rules (dangling comma, ending white space, two spaces) and then just have Prettier fix is when the commit happens.
https://github.com/prettier/prettier